app/User.php

```diff
@@ -95,6 +95,11 @@ public function commentVotes() {
         return $this->hasMany(CommentVote::class);
     }
 
+    /**
+     * Indicates if this user is flagged as an admin
+     *
+     * @return boolean
+     */
     public function isAdmin() {
         return $this->type === User::ADMIN_TYPE;
     }
```

Comment:
Add validate method

Reply:
What does it mean?

Reply:
I will not add validations to User. User's source code is created by the login middleware we use and that's not our problem